### ISSUE TITLE
rc7 does not start at all on macos

### ISSUE THREAD

**Issue by rgaudin** [kiwix/kiwix-hotspot#249]
In GitLab by @Kelson42 on Sep 12, 2018, 05:23

```
$ ./kiwix-plug_installer.app/Contents/MacOS/kiwix-plug_installer 

** (process:21350): WARNING **: 07:22:36.962: Failed to load shared library '@loader_path/libgdk-3.0.dylib' referenced by the typelib: dlopen(@loader_path/libgdk-3.0.dylib, 9): Library not loaded: @loader_path/libglib-2.0.0.dylib
  Referenced from: /Applications/kiwix-plug_installer.app/Contents/MacOS/libharfbuzz.0.dylib
  Reason: Incompatible library version: libharfbuzz.0.dylib requires version 5801.0.0 or later, but libglib-2.0.0.dylib provides version 5601.0.0
Traceback (most recent call last):
  File "__main__.py", line 6, in <module>
  File "runpy.py", line 208, in run_module
  File "runpy.py", line 85, in _run_code
  File "gui.py", line 29, in <module>
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 656, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 626, in _load_backward_compatible
  File "gi/importer.py", line 145, in load_module
  File "importlib/__init__.py", line 126, in import_module
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 656, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 626, in _load_backward_compatible
  File "gi/importer.py", line 146, in load_module
  File "gi/overrides/__init__.py", line 125, in load_overrides
  File "importlib/__init__.py", line 126, in import_module
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "/usr/local/lib/python3.6/site-packages/PyInstaller/loader/pyimod03_importers.py", line 631, in exec_module
  File "gi/overrides/Gdk.py", line 81, in <module>
  File "gi/overrides/__init__.py", line 202, in override
AssertionError
Failed to execute script __main__
Emmanuels-MacBook-Air:Applications kelson$ 

```

**Comment by rgaudin** [comment]
In GitLab by @rgaudin on Sep 12, 2018, 08:01

closed via commit c97fc7e0761712833cf42bd0532ec59c5e4aaa81

**Comment by rgaudin** [comment]
In GitLab by @rgaudin on Sep 12, 2018, 08:02

Fixed ; will wait for new master (integrating the missing wikiafrica icon in wikifundi) to publish rc8. Test build is [here](http://tmp.kiwix.org/ci/kiwix-plug_installer-macos-build.dmg).

**Comment by rgaudin** [comment]
In GitLab by @rgaudin on Sep 12, 2018, 11:52

reopened

**Comment by rgaudin** [comment]
In GitLab by @rgaudin on Sep 12, 2018, 13:52

closed via commit 70ef3c2bd8b082655e45992d796a1d0d90b2b215

**Comment by rgaudin** [comment]
In GitLab by @rgaudin on Sep 12, 2018, 13:52

closed via commit ca4670f0c7c7b536a5b71e279d8b85c1a1eff871
